prow/release-commit.sh

```diff
@@ -66,7 +66,7 @@ ${DEPENDENCIES:-$(cat <<EOD
     auto: deps
   pkg:
     git: https://github.com/istio/pkg
-    auto: modules
+    branch: master
```

Comment:
Should remove `pkg` entirely (L67-69). We will need the same change in istio/release-builder as a followup